contributors/devel/contributing-api-changes.md

```diff
@@ -0,0 +1,154 @@
+
+All changes to existing or new APIs are subject to the process outlined in this document.
+
+The following qualify as API changes:
+
+- Adding, removing, or modifying API types (e.g. through types.go, or by defining a new annotation)
+- Changing the version of an existing type (e.g. v1alpha1 to v1beta1)
+- Adding, removing, or modifying subresources for existing types. (e.g. /scale)
+- Material changes to how fields on existing types are interpreted / used - including, but not limited to
+  validation and defaulting of the fields.
+
+The following on their own do not qualify as API changes, but should follow the process defined by the related
+[special interest groups].
+
+- Adding or changing a command or flag to kubectl: See [contributing to SIG cli]
+- Building new command line tools: See [contributing to SIG cli]
+- Adding or changing a flag to kubelet: sig-node
+- Adding or changing a flag to apiserver: sig-api-machinery
+- Refactoring code
+- Building extensions
+
+## Life of an API change
+
+1. Start a discussion
+2. Write a short description
+3. Write a detailed design proposal in the [kubernetes/community] repo
+4. Schedule a design review
+5. File an issue in the [kubernetes/features] repo that links to the proposal
+6. Implement the proposal in the [kubernetes/kubernetes] repo
+7. Write user facing documentation on [kubernetes/kubernetes.github.io]
+
+### Start a discussion.
+
+Expected time for decision: 1-4 weeks (depending on the SIG and feature)
+
+Start a discussion with the relevant [special interest groups] that will [own] the changes.
+Depending on the SIG, this may be done via SIG meetings, email groups, slack channel, etc.
+
+### Write a short description
+
+Expected time for decision: 2 weeks
+
+Write a short description of the API changes by copying the [design summary template]
+into a new issue, and [@mention] @kubernetes/api-reviewers and the owning SIG reviewers
+group.
+
+Determine whether @kubernetes/api-reviewers is willing to accept a design proposal
+for your feature.  The answer could be "no, we don't want to solve this problem this way."
+or "no, this is something we want, but cannot commit the resources to reviewing it this release."
+
+### Write a detailed design proposal
+
+Expected time for initial feedback: 3-6 weeks
+
+Write a detailed design proposal PR using the [design proposal PR template].  [@mention]
+@kubernetes/api-reviewers and the appropriate SIGs on the PR and reference the original
+issue in the PR description.
+
+### Schedule a design review
+
+Expected time for next slot: 4-8 weeks
+
+Schedule a time for your design to be reviewed.
+
+Design approvers:
+- @smarterclayton
+- @bgrant0607
+- @thockin
+- @erictune
+
+### Implement the proposal
+
+Once the proposal has been accepted an merged, you can begin implementing the solution.
+
+## How to escalate
+
+Escalation should be done through the owning SIG.  If you need help getting attention, reach out to your
```

Comment:
Where do you think is the proper place for this to live?  I could create a document, but it will be constantly stale.  WDYT of comments in the code that would could `lint` for?